charts/remote-headless/templates/remote-headless.yaml

```diff
@@ -182,7 +182,7 @@ spec:
       name: {{ default "remote-headless" $.Values.nameOverwrite }}-data-{{ $index }}
     spec:
       accessModes:
-      - ReadWriteOnce
+      - {{ default "ReadWriteOnce" $.Values.storage.accessMode }}
```

Comment:
i think we may need default value for `$.Values.storage.accessMode` in `values.yaml` for all-in-one chart. error may occur for no existence of this value